scripts/limine.cfg

```diff
@@ -18,6 +18,24 @@ CMDLINE=bochs init.launch=kmscon plainfb.force=1
 MODULE_PATH=uuid://@ROOT_UUID@/boot/managarm/thor
 MODULE_PATH=uuid://@ROOT_UUID@/boot/managarm/initrd.cpio
 
+:managarm (Xorg, e9 output, plainfb)
```

Comment:
Is this needed for this PR or a remnant of our efforts to get pure X working? In the last case, we can remove it.